### PR TITLE
Remove geocoding callback from station model

### PR DESCRIPTION
**Pull request by jluria**
- Remove the after validation callback from stations, because they already have geographic coordinates
- Remove the location column from the stations table, because the addresses are already a part of the objects

### CODE REVIEW

**Comment by jluria** [comment]
@hrs I know this seems basic, but just wanted to check to see if there was anything I should be aware of. Code review, always?

File: app/models/station.rb [hunk 1]
```diff
@@ -1,4 +1,3 @@
 class Station < ActiveRecord::Base
-  reverse_geocoded_by :latitude, :longitude, address: :location
-  after_validation :reverse_geocode
```
Comment (hrs) [inline app/models/station.rb]: I'm not sure how `reverse_geocoded_by` works, but I'm guessing that—since you're removing it—it adds an `after_validation` callback of its own?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Bykr
+====
+
+Installation
```
Comment (hrs) [inline README.md]: Your documentation could use a bit of work. I might save this for a separate PR.

Comment (jluria) [inline README.md]: I'm actually working on this right now. I thought that documentation should not be its own PR because it's not a user/app/admin function. Should documentation be its own PR, then?

File: db/migrate/20150219194157_remove_location_from_stations.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class RemoveLocationFromStations < ActiveRecord::Migration
+  def change
+    remove_column :stations, :location, :string
```
Comment (hrs) [inline db/migrate/20150219194157_remove_location_from_stations.rb:3]: Is there any interesting information in the `location` column in production? I'm guessing there isn't, but if there _is_ you'll want to do something with it before dropping the column.

File: README.md (hunk 2 above)
Comment (hrs) [inline README.md]: Sure, I'd keep it separate. To look at it another way, this documentation doesn't have much to do with the "Remove geocoding callback from station model" PR.

File: app/models/station.rb (hunk 1 above)
Comment (jluria) [inline app/models/station.rb]: `reverse_geocoded_by` lets the Geocoder module know which attributes of the object to use with its included methods, but `after_validation` makes an api request, which I both don't need (because the station objects come with their own latlongs), and the requests bump up against the query limit many times in the process. Just seems unnecessary.

Comment (hrs) [inline app/models/station.rb]: Ah, okie-dokie, that totally makes sense. Just wanted to make sure!

**Comment by hrs** [comment]
Assuming there's nothing useful in the `location` column in prod, this looks good to me! :goat:it.

File: app/models/station.rb (hunk 1 above)
Comment (jluria) [inline app/models/station.rb]: The Geocoder module includes a method, `to_coordinates`, which takes an objects latlong attributes, and generates a geocoordinates array for each object, which other module methods use to make distance calculations, etc.

File: db/migrate/20150219194157_remove_location_from_stations.rb (hunk 3 above)
Comment (jluria) [inline db/migrate/20150219194157_remove_location_from_stations.rb:3]: `location` ends up being the same information provided in each station object's `address` column. Realistically, I guess I should include a method in the rake task to make sure no `address` columns are `null`.
